can_messages.asciipb

```diff
@@ -1,294 +1,352 @@
+# Message Definitions in plaintext are on the wiki at:
+# https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/50003973/CAN+Message+Definitions
+#
+# If you are making changes to this file please update the corresponding entry
+# on the wiki. If you need to add a new message use a reasonable
+# reserved ID. The higher ID the lower the priority. Generally 
+# - 0-13: Critical messages (have ACK)
+# - 14-30: Actionable messages (trigger a change in another system)
+# - 30-63: Data messages (usually not actionable by an onboard device)
+
 msg {
-  id:0
+  id: 0
+  source: PLUTUS
   msg_name: "bps fault"
```

Comment:
We might want to have a "message name" and "readable name" since we have a few messages that are getting pretty long (like `CAN_MESSAGE_MOTOR_CONTROLLER_ERROR_AND_LIMITS`)

Reply:
Done. Good Idea.